src/lib/contracts/CheckoutPopupOptions.ts

```diff
@@ -212,6 +217,114 @@ export interface CheckoutPopupParams {
         ctx: string;
         token: string;
     };
+    /**
+     * Specify the layout of the form on a larger screen.
+     * @default null
+     */
+    layout?: 'vertical' | 'horizontal' | null;
```

Comment:
If you like adding line space after every property (I do like it), then please do it for every property in this file (just for consistency).